packages/mobile/src/fiatExchanges/MoonPay.tsx

```diff
@@ -39,11 +48,11 @@ function FiatExchangeWeb() {
           Accept: 'application/json',
           'Content-Type': 'application/json',
         },
-        // TODO: Add amount here
         body: JSON.stringify({
           currency: celoCurrencyCode,
           address: account,
           fiatCurrency: localCurrencyCode,
+          ...(localAmount && { fiatAmount: localAmount.toString() }),
```

Comment:
Nit, would `fiatAmount: localAmount.?toString()` work? I don't know what stringify does when given null

Reply:
I've tested, it should do the same, what I did. Will replace it, thanks!